File: content/Учёба/Практика/2.1.2026/Практика/MATLAB/1.m
x = -5:0.1:5;
y = 4 .* x.^2 - 2 .* x + 6;
figure;
plot(x, y, 'y-.', 'LineWidth', 2);
set(gca, 'Color', [0.3 0.3 0.3]); 
grid on;
title('f(x) = 4x^2 - 2x + 6');
xlabel('X');
ylabel('Y');

File: content/Учёба/Практика/3.2.2026/Практика/PYTHON/1.py
import numpy as np
import matplotlib.pyplot as plt

# Область определения функции
x = np.linspace(-5, 5, 400)
# Значения функции (векторизованная операция numpy)
y = 4 * x**2 - 2 * x + 6
# График
fig, ax = plt.subplots(figsize=(8, 6))
ax.set_facecolor('#4d4d4d')
ax.plot(x, y, color='yellow', linestyle='-.', linewidth=2, label='f(x) = 4x² - 2x + 6')
ax.grid(True, color='gray', linestyle='--', linewidth=0.5)
ax.set_xlabel('X', fontsize=12)
ax.set_ylabel('Y', fontsize=12)
ax.legend()

plt.show()